modules/cdp-engine/src/tests.rs

```diff
@@ -416,7 +416,7 @@ fn expand_position_collateral_work() {
 
 		assert_noop!(
 			CDPEngineModule::expand_position_collateral(&ALICE, DOT, 0, 1),
-			Error::<Runtime>::CannotSwap
+			DispatchError::CannotLookup
```

Comment:
This breaking change doesn't make sense to me. The reason of swap failure could be different, instead of just lookup errors.

Reply:
Add another Error type `CannotLookup` in dex module, now when invoke `swap_supply_with_best_price_swap_path`/`swap_with_best_price`, the error is CannotLookup defined in dex module.